Makefile

```diff
@@ -452,6 +476,10 @@ help:
 	@echo "  MODULE_LOAD=1             Before compiling or running, module load is called"
 	@echo "  ADD_BATCH_SCHED=1         Add the jsrun command before the execution of the running script to send it to a compute node"
 	@echo "  NO_OFFLOADING=1           Turn off offloading"
+	@echo "  LINK_OMPVV_LIB=1          Link the OMPVV static library (found in sollve_vv/ompvv)"
```

Comment:
What happens if we just always link it?

Reply:
I think this would work, since you have to include the header file anyways if you want to use it. I'll make this change.